### PR TITLE
[184651] count messages in all mailboxes

### PR DESCRIPTION
**Pull request by aDaichiOta**
https://acsmine.tok.access-company.com/redmine/issues/184651

### CODE REVIEW

File: test/core/util/vm_test.exs [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Copyright(c) 2015-2019 ACCESS CO., LTD. All rights reserved.
+
+defmodule AntikytheraCore.VmTest do
+  use Croma.TestCase
+  alias AntikytheraCore.Vm
+
+  test "count_messages_in_all_mailboxes/0" do
+    count = Vm.count_messages_in_all_mailboxes
```
Comment (irisTa56) [inline test/core/util/vm_test.exs]:
```suggestion
    count = Vm.count_messages_in_all_mailboxes()
```

**Comment by irisTa56** [comment]
LGTM except for one comment!

File: test/core/util/vm_test.exs (hunk 1 above)
Comment (aDaichiOta) [reply test/core/util/vm_test.exs]: thanks

**Comment by aMasakiTakahashi** [comment]
LGTM!

**Comment by ttokit** [comment]
LGTM!

**Pull request merged**